packages/pwa-kit-react-sdk/package.json

```diff
@@ -46,7 +46,8 @@
     "react-ssr-prepass": "^1.5.0",
     "react-uid": "^2.3.2",
     "serialize-javascript": "^6.0.2",
-    "svg-sprite-loader": "^6.0.11"
+    "svg-sprite-loader": "^6.0.11",
+    "winston": "^3.13.0"
```

Comment:
How much increase in size for the template after installing this one?

Reply:
<img width="1696" alt="Screenshot 2024-05-16 at 4 46 57 PM" src="https://github.com/SalesforceCommerceCloud/pwa-kit/assets/6024102/145deb84-adef-405b-9039-d25280aaa367">

wisnston gzipped: 8.14 Kb
winston-transport gzipped: 1.13Kb
Total build/ssr.js file increase with winston is = 9.27 Kb